### PR TITLE
Importing bls key from hex

### PR DESCRIPTION
**Pull request by shrimalmadhur**
Fixes | Closes | Resolves #

support Importing a key from hex private key

## Changes:
- _List the changes introduced by this PR more in detail here_

## Types of changes

_Leave on the following list the types of changes introduced by this PR and remove
the ones that don't apply. Please also remove this line._

- New feature (non-breaking change which adds functionality)

## Testing

**Requires testing** Yes/No

**In case you checked yes, did you write tests?** Yes/No

**Comments about testing , should you have some** (optional)

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by AntiD2ta — APPROVED** [review]
LGTM!

I strongly suggest adding regression tests for this code soon.

**Comment by shrimalmadhur** [comment]
> LGTM!
> 
> I strongly suggest adding regression tests for this code soon.

Just added!

File: cli/operator/keys/import_test.go [hunk 1]
```diff
@@ -110,6 +113,23 @@ func TestImportCmd(t *testing.T) {
...
+			args: []string{"--key-type", "bls", "test", "0xfe198b992d97545b3b0174f026f781039f167c13f6d0ce9f511d0d2e973b7f02"},
+			err:  nil,
+			promptMock: func(p *prompterMock.MockPrompter) {
+				p.EXPECT().InputHiddenString(gomock.Any(), gomock.Any(), gomock.Any()).Return("", nil)
+			},
+			expectedPrivKey: "5491383829988096583828972342810831790467090979842721151380259607665538989821",
+			keyPath:         filepath.Join(homePath, OperatorKeystoreSubFolder, "/test.bls.key.json"),
+		},
+		{
+			name:    "valid bls key import for hex key",
```
Comment (AntiD2ta) [inline cli/operator/keys/import_test.go]:
```suggestion
			name:    "invalid bls key import for hex key",
```

Comment (shrimalmadhur) [reply cli/operator/keys/import_test.go]: oo good catch thanks!

**Pull request merged**